godtools/App/Features/ToolDetails/Domain/GetToolVersionsUseCase/GetToolVersionsUseCase.swift

```diff
@@ -34,14 +38,69 @@ class GetToolVersionsUseCase {
         })
     }
     
-    private func getToolVersion(resource: ResourceModel) -> ToolVersion {
+    private func getToolVersion(resource: ResourceModel) -> ToolVersionDomainModel {
         
-        return ToolVersion(
-            id: UUID().uuidString,
+        let toolLanguages: [ToolLanguageModel] = getToolLanguagesUseCase.getToolLanguages(resource: resource)
+        let toolLanguagesIds: [String] = toolLanguages.map({$0.id})
+                
+        let name: String
+        let languageBundle: Bundle
+        
+        // TODO: Another place that needs to be completed in GT-1625. ~Levi
```

Comment:
👍

Reply:
Hey @rachaelblue yep circle selection will be in another ticket